src/table/column.rs

```diff
@@ -584,6 +585,18 @@ impl ColumnDef {
         self
     }
 
+    /// Sets the column as generated with SimpleExpr
+    pub fn generated<T>(&mut self, expr: T, stored: bool) -> &mut Self
```

Comment:
May be a doc test like above ? https://github.com/tooboredtocode/sea-query/blob/b6adc4b0f18fa24fafa6ac543bf49a4da190758e/src/table/column.rs#L246